### PR TITLE
chore: polish multi-agent example

### PR DESCRIPTION
**Pull request by huangjeff5**
Checklist (if applicable):
- [x] PR title is following https://www.conventionalcommits.org/en/v1.0.0/
- [x] Tested (manually, unit tested, etc.)
- [x] Docs updated (updated docs or a docs bug required)

### CODE REVIEW

File: samples/js-schoolAgent/src/routingAgent.ts [hunk 1]
```diff
@@ -16,16 +16,17 @@
 
 import { attendanceAgent } from './attendanceAgent';
 import { ai } from './genkit';
+import { gradesAgent } from './gradesAgent';
 import { searchEvents, upcomingHolidays } from './tools.js';
 
-export const infoAgent = ai.definePrompt(
+export const routingAgent = ai.definePrompt(
```
Comment (pavelgj) [inline samples/js-schoolAgent/src/routingAgent.ts:22]: I think I prefer `infoAgent`....

File: samples/js-schoolAgent/README.md [hunk 2]
```diff
@@ -1,11 +1,104 @@
...
+
+The agent uses a multi-agent architecture:
+
+- Routing Agent: Acts as the main entry point and router, handling general queries while delegating specialized requests to appropriate agents
+- Attendance Agent: Specialized agent focused on absence and tardy reporting
+- Grades Agent: Manages academic performance queries, grade reports, and transcript requests
+
+Example queries:
+
+- "Mark Evenlyn Smith as absent please"
```
Comment (pavelgj) [inline samples/js-schoolAgent/README.md:90]:
```suggestion
- "Evelyn will be late today"
```

File: samples/js-schoolAgent/src/attendanceAgent.ts [hunk 3]
```diff
@@ -16,16 +16,19 @@
...
-    description:
-      'transfer to this agent when the user asks questions about attendance-related concerns like tardies or absences. do not mention that you are transferring, just do it',
-    tools: [reportAbsence, reportTardy],
+    name: `${specialization}Agent`,
+    description: agentDescription(specialization, tools),
+    tools,
   },
   ` {{ role "system"}}
-  You are Bell, a helpful attendance assistance agent for Sparkyville High School. A parent has been referred to you to handle an attendance-related concern. Use the tools available to you to assist the parent.
+${agentPrompt(specialization)}
```
Comment (pavelgj) [inline samples/js-schoolAgent/src/attendanceAgent.ts:31]: This makes it hard to see what this agent does at a glance. I think I little duplication is OK. It makes customization easier.

**Review by pavelgj — APPROVED** [review]

**Review by peterfriese — CHANGES_REQUESTED** [review]
This is more of a walkthrough of an existing code sample, and not a hands-on codelab. 

While there is nothing wrong with a walkthrough, I would prefer giving people a hand-on experience of creating an agent system themselves, to show them how it's really done.

**Comment by huangjeff5** [comment]
> This is more of a walkthrough of an existing code sample, and not a hands-on codelab.
> 
> While there is nothing wrong with a walkthrough, I would prefer giving people a hand-on experience of creating an agent system themselves, to show them how it's really done.

Makes sense. I think it will take some more work and iteration than I initially thought, so I'm going to work on this and try to simplify it and make it more hands-on, and make a separate PR.

**Review by peterfriese — APPROVED** [review]
OK to merge as a sample.

**Pull request merged**